certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "1.15.0",
+		"title": "RAPID Platform (V1)",
+		"description": "The RAPID Platform connector allows you to interact with your RAPID sites to help automate your business.",
+		"license": {
+			"name": "RAPID Platform License"
+		},
+		"contact":{
+			"name": "RAPID Support",
+			"url": "http://www.rapidplatform.com.au",
+			"email": "support@rapidplatform.com.au"
+		}
+	},
+	"host": "api-test.rapidplatform.com",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/hooks": {
+			"x-ms-notification-content": {
+				"description": "Register a webhook on your RAPID site.",
+				"schema": {}
+			},
+			"post": {
+				"x-ms-trigger": "single",
+				"operationId": "ListHookTrigger",
+				"description": "Creates a webhook for when something happens in your RAPID site.",
+				"summary": "Trigger when something happens to an item in your RAPID site",
```

Comment:
@MattFranklin-Rapid If it is trigger, it would always display under "Triggers" tab, So can you please delete `Trigger` word from the Summary value and start the value with "When ..."

Reply:
Updated summary to start with "When"